tests/locator_test.go

```diff
@@ -169,3 +169,28 @@ func TestLocatorFill(t *testing.T) {
 		require.Panics(t, func() { link.Fill(value, nil) }, "should not select multiple elements")
 	})
 }
+
+func TestLocatorFocus(t *testing.T) {
+	tb := newTestBrowser(t, withFileServer())
```

Comment:
I think we'll need to start adding `t.Parallel()` to at least these top-level tests. One or two without it is OK, but with the amount we're adding for `Locator`, it's increasing the total test runtime enough to be noticeable.

On `v0.3.0`, `GOMAXPROCS=3 go test -p 3 -v -race -count=1 ./...` finishes in 41.9s on my machine. The same command on `main` (before this was merged) takes 48.2s.

I know we run the risk of flakiness with more tests running in parallel, but I'd rather run that risk than incur the duration penalty every time I run the test suite.

Reply:
See: https://github.com/grafana/xk6-browser/pull/389